travis/after-success-gradle

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -eux
+
+bash ./deploy_snapshot.sh
```

Comment:
Remove `bash `, so that the `deploy_snapshot.sh` script can choose its own interpreter (and interpreter flags) via an initial `#!` line.